tests/checks/common.py

```diff
@@ -14,6 +14,7 @@
 from checks import AgentCheck
 from config import get_checksd_path
 from util import get_os, get_hostname
+from utils.debug import get_check  # noqa
```

Comment:
Could you run a `sed` instead ? I think it's only used in old tests. Also, do you know the difference between `load_check` and `get_check` ? Because `load_check` seems to be used in `AgentCheckTest`.